doc/geany.txt

```diff
@@ -2246,6 +2246,10 @@ Single quotes ' '
 Double quotes " "
     Auto-close double quotes when typing an opening one
 
+For a more fluent writing one can also activate that Geany checks if the
```

Comment:
Not sure "fluent" will be correctly understood, maybe just say to avoid having to move your typing hand off the keyboard to the arrow keys as you explained in the issue description.

And apparently "Fluent" is a type of coding paradigm in C#.